src/jit/emitxarch.cpp

```diff
@@ -12213,6 +12267,16 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             sz = emitSizeOfInsDsc(id);
             break;
 
+        case IF_AWR_RRD_CNS:
+        case IF_MWR_RRD_CNS:
```

Comment:
These should be handled separately as well. `AWR` goes through `emitOutputAM`, but `MWR` needs to go through `emitOutputCV`. The functions also deal with regcode slightly differently.

Reply:
This PR just add `emitIns_AR_R_I`, shall I remove `IF_MWR_RRD_CNS`?

Reply:
I think we need/want both to ensure all the same code exists as for the other cases. 

`emitMapFmtAtoM` specifically provides a mapping from one to the other.